validator/rpc/wallet.go

```diff
@@ -263,7 +264,7 @@ func (s *Server) initializeWallet(ctx context.Context, cfg *wallet.Config) error
 	}
 
 	s.walletInitialized = true
-	km, err := w.InitializeKeymanager(ctx)
+	km, err := w.InitializeKeymanager(ctx, iface.InitKeymanagerConfig{ListenForChanges: false})
```

Comment:
I think this should be true, because then this keymanager is sent to the validator client service over an event feed